services/viva-ms/lambdas/generateRecurringCaseHtml.js

```diff
@@ -29,6 +29,36 @@ handlebars.registerHelper({
   },
 });
 
+async function getClosedUserCases(primaryKey) {
+  const dynamoDbQueryCasesParams = {
+    TableName: config.cases.tableName,
+    KeyConditionExpression: 'PK = :pk and begins_with(SK, :sk)',
+    FilterExpression: 'begins_with(#status.#type, :statusTypeClosed)',
+    ExpressionAttributeNames: {
+      '#status': 'status',
+      '#type': 'type',
+    },
+    ExpressionAttributeValues: {
+      ':pk': primaryKey,
+      ':sk': 'CASE#',
+      ':statusTypeClosed': 'closed',
+    },
+  };
+  return await to(dynamoDb.call('query', dynamoDbQueryCasesParams));
+}
+
+function setChangedCaseAnswerValues(currentAnswerList, previousAnswerList) {
```

Comment:
usually it isn't good practise to modify/reassign input function parameters.

Reply:
I have fixed this now. Not really reassigning/modifying though. It could be debated if adding to an existing array is breaking the rule of immutabillity.